spec/mailers/abuse_mailer_spec.rb

```diff
@@ -87,11 +87,44 @@
    it "body text contains the comment" do
      expect(mail).to have_body_text(/#{report.comment}/)
    end
-
+   
    it "has the url of the page with abuse" do
      expect(mail.html_part).to have_body_text(report.url)
    end
   end
 
+  
+  context "abuse_reports contains creator and title if reporting a work" do
+   let(:report) {create(:abuse_report, url: "http://archiveofourown.org/works/3753052?view_full_work=true")}
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing inside {.<br>Space missing inside }.